core/templates/dev/head/css/oppia.css

```diff
@@ -3928,10 +3928,17 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  height: 70px;
+  line-height: 1.26;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
   padding: 12px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3; /* Truncating to 3 lines*/
+  max-height: 4.41176471em; /*1.26 (line-height) * 3 / 0.85 (font-size) */
```

Comment:
One additional consideration I did not think of for this calculation: characters such as y, j, and g that go beneath the baseline will not display fully because this cuts off _at_ the baseline. With that in mind, we should add perhaps ~25% of the line-height times the font-size.

/* This is calculated as 1.26 (line-height) * 3 lines / 0.85 (font-size). We add an additional 1.26 (line-height) * 0.85 (font-size) * 0.25 to account for letters that fall beneath the baseline. */
max-height: 4.68;